.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
+  },
+  "editor.formatOnType": false,
+  "editor.formatOnPaste": true,
+  "editor.formatOnSave": true,
```

Comment:
:( I don't like these two, I like formatting when I want, not being forced when I save